src/scss/customOverrides.scss

```diff
@@ -5,23 +5,23 @@
 // hide time on site and number of venues visited box (when badges are turned on)
 .Badges {
   &__visits {
-    display: none;
+    display: none !important;
   }
   &__visit {
-    display: none;
+    display: none !important;
   }
 }
```

Comment:
These still seem to use `!important`, which goes against what you suggested to be the case in https://github.com/sparkletown/sparkle/pull/1755#issuecomment-877652052 ? Whys that?

Reply:
fixed in d414cb523b6cf736e1693468c5beeee0686bb8b4

Reply:
fixed in c129a927c7a43aec64d1228a2bf25f6194528f40